### PR TITLE
[DOCS] Update HTTP and Kafka components.

### PR DESCRIPTION
**Pull request by mmajerni**

### CODE REVIEW

File: components/camel-kafka/src/main/docs/kafka-component.adoc [hunk 1]
```diff
@@ -228,7 +228,7 @@ camelContext.addRoutes(new RouteBuilder() {
...
 Each repository instance that uses the topic (e.g. typically on different machines running in parallel) controls its own consumer group, so in a cluster of 10 Camel processes using the same topic each will control its own offset.
-On startup, the instance subscribes to the topic and rewinds the offset to the beginning, rebuilding the cache to the latest state. The cache will not be considered warmed up until one poll of `pollDurationMs` in length returns 0 records. Startup will not be completed until either the cache has warmed up, or 30 seconds go by; if the latter happens the idempotent repository may be in an inconsistent state until its consumer catches up to the end of the topic.
+On startup, the instance subscribes to the topic, rewinds the offset to the beginning and rebuilding the cache to the latest state. The cache will not be considered warmed up until one poll of `pollDurationMs` in length returns 0 records. Startup will not be completed until either the cache has warmed up, or 30 seconds go by; if the latter happens the idempotent repository may be in an inconsistent state until its consumer catches up to the end of the topic.
```
Comment (orpiske) [inline components/camel-kafka/src/main/docs/kafka-component.adoc:231]: I think this should be:

```On startup, the instance subscribes to the topic, rewinds the offset to the beginning and rebuilds the cache to the latest state.
```

**Review by orpiske — CHANGES_REQUESTED** [review]
Thanks for the contribution. Overall it LGTM, but there is a small typo in one of the corrections.

File: components/camel-kafka/src/main/docs/kafka-component.adoc (hunk 1 above)
Comment (mmajerni) [reply components/camel-kafka/src/main/docs/kafka-component.adoc]: Fixed. Thanks, I missed that.

**Review by orpiske — APPROVED** [review]
Thanks for your contribution!

**Review by davsclaus — APPROVED** [review]

**Pull request merged**